### ISSUE TITLE
Generated test code breaks when using the no-context option

### ISSUE THREAD

**Issue by preslavrachev** [volatiletech/sqlboiler#1239]
When applying the option `no-context=true` as part of the `TOML` configuration file, the generated **test** code comes with compile time errors. I was able to attribute that to the new additions (in v4.14.0):

```
AddUserHook(boil.AfterSelectHook, func(e boil.ContextExecutor, o *User) error {
    ranAfterSelectHook = true
    return nil
})
```
The generated code above won't compile, because the function spec needs a `boil.Executor` and not `boil.ContextExecutor`. When removing the `no-context` configuration option, everything works as it is supposed to. Unfortunately, requiring a context in all of our existing code will be overkill.

Is there a quick way I can solve this?

### What version of SQLBoiler are you using (`sqlboiler --version`)?

v4.14.0

### What is your database and version (eg. Postgresql 10)

PostgreSQL 14

### If this happened at generation time what was the full SQLBoiler command you used to generate your models? (if not applicable leave blank)

``` 
sqlboiler psql --wipe
```

**Comment by stephenafamo** [comment]
Should be fixed in `v4.14.1`

**Comment by preslavrachev** [comment]
Confirmed, works as expected. Thanks!
